### ISSUE TITLE
abbreviations not working 

### ISSUE THREAD

**Issue by bandrel** [glitchassassin/keyboard#2]
The other items in the examples work, but i can't seem to get ```keyboard.add_abbreviation('@@', 'my.long.email@example.com')``` to work.

**Comment by glitchassassin** [comment]
Finally getting back to this.

`@@` does not work. I tried `ii` and got an error message. Digging deeper.

**Comment by glitchassassin** [comment]
The _os_keybord is missing a required method: `type_unicode()`.

**Comment by glitchassassin** [comment]
Also, the "backspace" key was not defined. Added it as an alias for "delete".

**Comment by glitchassassin** [comment]
`@@` does not work, but `ii`, `II`, and `22` all work.

**Comment by glitchassassin** [comment]
There are a couple things going on here.

I just tested the latest version of `keyboard` on Windows and confirmed that it suffers the same problem: the word listener depends upon upper() to capitalize lowercase characters, which works for `ii` and `II` but not `22` and `@@`.

There was also an issue with the Mac fork where modifier keys (shift, alt, etc.) were not being detected by the key listener, which caused the word listener to ignore capitalized words. So while `ii` and `22` were working, `II` was actually not working on Mac.

I've fixed the issue with the key listener, so shift keys will now be correctly reported. I also added "right command" to the key list.
